bip-0341.mediawiki

```diff
@@ -106,7 +106,7 @@ If the parameters take acceptable values, the message is the concatenation of th
 ** If the ''hash_type & 0x80'' does not equal <code>SIGHASH_ANYONECANPAY</code>:
 *** ''sha_prevouts'' (32): the SHA256 of the serialization of all input outpoints.
 *** ''sha_amounts'' (32): the SHA256 of the serialization of all spent output amounts.
-*** ''sha_scriptpubkeys'' (32): the SHA256 of the serialization of all spent output ''scriptPubKey''s.
+*** ''sha_scriptpubkeys'' (32): the SHA256 of the serialization of all spent output ''(compact_size(size of scriptPubKey) || scriptPubKey)''s.
```

Comment:
sha_scriptpubkeys is the hash of the **serializations** of all spent scriptPubKeys. The serialization of a scriptPubKey is already (compact_size(size of scriptPubKey) || scriptPubKey).  I know it may not be super clear but I don't think we should modify it. If it is necessary we should rather put a note somewhere to remember what the serialization of a scriptPubKey

Reply:
True, I'll adjust the wording to be correct. The reason this part was confusing as written is that later in the description of SigMsg the annex is specifically called out as having its `compact_size` prepended to it. I think there should be consistency between this and that.

Reply:
In the description below for `scriptPubKey` it says "serialized as script inside <code>CTxOut</code>". This is identical to your suggestion but it would be better to be consistent.

Reply:
Thanks. Changed to use the common wording.